metadata.json

```diff
@@ -11,7 +11,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.13.0 < 9.0.0"
+      "version_requirement": ">= 9.4.0 < 10.0.0"
```

Comment:
If you don't make code changes, please don't bump the lower bound. Why specifically 9.4.0 as the lower bound?